vnext/Desktop.ABITests/React.Windows.Desktop.ABITests.vcxproj

```diff
@@ -137,9 +137,10 @@
   <ItemGroup>
     <ClInclude Include="MessageQueueShim.h" />
   </ItemGroup>
-  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
+  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" Condition="$(NoCppWinRT) == ''" />
```

Comment:
Can we just change the normal desktop build to use NoCppWinRT?  Maintaining two sets of build rules, and def files is a giant pain.